modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java

```diff
@@ -94,4 +174,9 @@ protected boolean isValidExtension(String fileName) {
 		return false;
 	}
 
+	private static final Pattern _commentsPattern = Pattern.compile(
+		"(<%--([0-9]*[\\*\\w\\s\\-\\:\\(\\)\\.\\,\\/]+)--%>)");
+	private static final Pattern _directivePattern = Pattern.compile(
```

Comment:
@TamyrisTorres, could you tell me why you think the pattern is named 'directive'?

Reply:
In my search, I saw that in .jsp call "include directive".

Reply:
As we discussed offline, we should rename it to _includesPattern

Reply:
Change for  '_includePattern'